dts/ember/0000-sparx5-135-emmc/device-tree.dts

```diff
@@ -28,7 +28,7 @@
 &spi0 {
 	pinctrl-0 = <&si2_pins>;
 	pinctrl-names = "default";
-	status = "okay";
+	status = "okay"
```

Comment:
Why drop the semicolon?

Reply:
That was the only thing that stuck out for me, I'll leave the rest of the dts review to @joazet